.github/workflows/pull_request.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.19','1.20']
+        go: ['1.20','1.21']
```

Comment:
u need to modify `go: ['1.20']`in the other yml files to be `go: ['1.21']`

Reply:
no, we keep using 1.20 as the main one, since this is what's still used atm for image builds. 1.21 is like a "preview" atm